backend/apps/accounts/views.py

```
@@ -583,642 +103,163 @@ def post(self, request):
-class LogoutView(APIView):
+class ChangePasswordView(APIView):
     """
-    Accepts a refresh token in the request body and adds it to the blacklist.
-    Requires user to be authenticated.
+    POST /api/auth/change-password/
+
+    Change user password and invalidate all existing JWT tokens.
+    Rate-limited to 5 requests/minute per user.
     """
 
     permission_classes = [IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)